packages/wxt/src/core/create-server.ts

```diff
@@ -130,7 +131,9 @@ export async function createServer(
   });
 
   // Listen for file changes and reload different parts of the extension accordingly
-  const reloadOnChange = createFileReloader(server);
+  const reloadOnChange = debounce(createFileReloader(server), 3e2, {
+    leading: true,
+  });
```

Comment:
I'm expecting unrealistic and excessive changes by machine. Not sure if the wait-time should be shorter or longer.... 🤔

Reply:
300ms still causes the double reloads issue, I have to bump it to at least 3s

Reply:
@pnd280 I turned off trailing, so 300ms might have been OK. Please check. If it's still NG, please send us a captured video of your screen.